packages/pyodide-kernel/src/worker.ts

```diff
@@ -92,7 +92,15 @@ export class PyodideRemoteKernel {
   protected async initKernel(options: IPyodideWorkerKernel.IOptions): Promise<void> {
     const preloaded = (options.loadPyodideOptions || {}).packages || [];
 
-    const toLoad = ['ssl', 'sqlite3', 'ipykernel', 'comm', 'pyodide_kernel', 'ipython'];
+    const toLoad = [
+      'ssl',
+      'sqlite3',
+      'ipykernel',
+      'comm',
+      'pyodide_kernel',
+      'jedi',
```

Comment:
I guess this is the main drawback for adding it back, as it may increase the kernel startup time a little.

But having better completions also makes it worth it.